spec/models/hmcts_common_platform/application_summary_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe HmctsCommonPlatform::ApplicationSummary, type: :model do
+  let(:hearing_summary) { described_class.new(data) }
+
+  context "with all fields" do
+    let(:data) { JSON.parse(file_fixture("application_summary/all_fields.json").read) }
+
+    it "matches the HMCTS Common Platform schema" do
+      expect(data).to match_json_schema(:application_summary)
+    end
+
+    it { expect(hearing_summary.id).to eql("6cd6494e-5409-420a-bd0b-f589dbb2466b") }
```

Comment:
Maybe it's worth adding a spec that verifies the output of `summary.to_json`

Reply:
yes! that would allow spot the error with `receivedDate` instead of `received_date`
Done!